### ISSUE TITLE
continuous deploy to public prod-like environment so people can try out the app

### ISSUE THREAD

**Issue by gobengo** [regensunite/regenjournal#4]

**Comment by gobengo** [comment]
looks like current domain is managed by vercel.
personally id rather use cloudflare workers, aws, or some kubernetes
but whatever.

**Comment by xdamman** [comment]
It's already set up to automatically deploy when you push to the main branch on https://regenjournal.xyz via vercel

**Comment by gobengo** [comment]
@xdamman got it. it 404s now just because we haven't defined a route?
https://regenjournal.xyz/

(i gotta relearn next.js)

**Comment by xdamman** [comment]
No. It’s because there is an error in the build process because of typescript errors (works fine in dev). Try npm run build to see the errors (npm run dev works fine).
